languages/python/oso/tests/test_polar_roles.py

```diff
@@ -56,9 +54,8 @@ def __repr__(self):
 class Issue(Base):  # type: ignore
     __tablename__ = "issues"
 
-    issue_id = Column(Integer, primary_key=True)
-    name = Column(String(256))
-    repo_id = Column(Integer, ForeignKey("repos.repo_id"))
+    name = Column(String(256), primary_key=True)
+    repo_name = Column(String(256), ForeignKey("repos.name"))
```

Comment:
Let's pick a convention for defining these GitClub classes and stick with it (used in GitClub app + multiple tests). I'm assuming this is the naming used in the GitClub app, so we can just refer to that as the template from now on?

Reply:
Not sure if this matches GitClub; the changes here are just simplifying the fixtures we copied from `sam/polar-roles` to remove columns that the tests weren't actually hitting.